src/main/java/hudson/remoting/MimicException.java

```diff
@@ -29,7 +33,8 @@ public String toString() {
         return (message != null) ? (s + ": " + message) : s;
     }
 
-    public static Throwable make(Channel ch, Throwable cause) {
+    @Nullable
```

Comment:
`@CheckForNull` does not force you to check the return value for null, rather it says that the return value is likely to be `null`... whereas `@Nullable` says that the caller of the method is supposed to know whether they need to check for null or not.

Either mark the method `@CheckForNull` or mark the parameter `@Nullable` if you want my blessing... Ignore if  you want to go ahead with maintainer privilidges